.gitignore

```diff
@@ -61,9 +61,6 @@ test/*.xml
 # Emacs backup files
 *~
 
-# Automatically generated when needed
```

Comment:
cc @jrajahalme give that you add this, you might have had reasons for this to be added.

Reply:
@aanm This was added due to the `.dockerignore` being automatically created for the Envoy proxy docker builds. Now that proxy is in a separate repo, this can be safely removed.